sdk/apimanagement/Azure.ResourceManager.ApiManagement/Azure.ResourceManager.ApiManagement.sln

```diff
@@ -9,6 +9,8 @@ Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Azure.ResourceManager.ApiMa
 EndProject
 Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Azure.ResourceManager.ApiManagement.Samples", "samples\Azure.ResourceManager.ApiManagement.Samples.csproj", "{B20CA0CA-F4A7-4B38-AF96-4C4F3D57D51E}"
 EndProject
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Azure.Core.TestFramework", "..\..\core\Azure.Core.TestFramework\src\Azure.Core.TestFramework.csproj", "{262B4D5F-A8FA-4416-9614-7800DDCC7BC8}"
```

Comment:
Do not add project reference of test framework if it's not there before.

Reply:
fixed.